Added script to minify CSS files

axelmortimer opened the pull request.

philsinatra commented:
1. Your README instructions are incomplete. I can't run `node index.js` before doing an `npm install` to get your dependencies.
1. Your `package.json` file has a parse error.
1. Ideally you should be checking out a separate feature branch for each challenge/task. As it is now, you have everything in your _master_ branch which is why everything is piling up on top.

## Parse Error Details

```bash
adm [axelmortimer-master]$ npm install
npm ERR! file /IDM-Git-Practice/challenge_2/adm/package.json
npm ERR! code EJSONPARSE
npm ERR! JSON.parse Failed to parse json
npm ERR! JSON.parse Unexpected token } in JSON at position 251 while parsing near '...-css": "^4.1.11",
npm ERR! JSON.parse   }
npm ERR! JSON.parse }
npm ERR! JSON.parse '
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.
```

axelmortimer commented:
I believe I just fixed the package.json error (at least it works when I run it on my side) and updated the README.md with better instructions. Do I need to submit another pull request or does the process still happen here?
